valuetransfer/src/main/java/nl/tudelft/trustchain/valuetransfer/dialogs/GatewayWarningDialog.kt

```diff
@@ -0,0 +1,43 @@
+package nl.tudelft.trustchain.valuetransfer.dialogs
+
+import android.os.Build
+import android.os.Bundle
+import android.widget.*
+import androidx.annotation.RequiresApi
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import nl.tudelft.trustchain.valuetransfer.R
+import nl.tudelft.trustchain.valuetransfer.ui.VTDialogFragment
+import nl.tudelft.trustchain.valuetransfer.util.*
+
+class GatewayWarningDialog : VTDialogFragment() {
+
+    @RequiresApi(Build.VERSION_CODES.O)
```

Comment:
What happens on older devices?

Reply:
We've checked and this check does not give any warnings so it has been removed.